packages/components-js/public/index.html

```diff
@@ -173,7 +182,7 @@
               /(<(?:my-prefix-)?p-[a-z-]+[\S\s]*?)>/g, // tweak components
               (m, g1) => g1.includes('theme') ? g1.replace(/theme="[a-z]+"/, `theme="${theme}"`) : `${g1} theme="${theme}">`
             )
-            .replace(/playground light/g, `playground ${theme}`); // tweak playgrounds
+            .replace(/(?<!\.)(playground)(?!--)(?:\slight|\sdark)?/g, `$1 ${theme}`); // tweak playgrounds
```

Comment:
`\s` includes 
> A space character
A tab character
A carriage return character
A new line character
A vertical tab character
A form feed character

So having a single ` ` space would be more precise. 
What do we need the `(?!--)` part for? If really needed, a comment would be helpful.